source/service_test.go

```diff
@@ -441,7 +451,7 @@ func BenchmarkServiceEndpoints(b *testing.B) {
 	_, err := kubernetes.CoreV1().Services(service.Namespace).Create(service)
 	require.NoError(b, err)
 
-	client, err := NewServiceSource(kubernetes, v1.NamespaceAll, "", "")
+	client, _ := NewServiceSource(kubernetes, "", "", "")
```

Comment:
let's leave the constant here. And let's also keep the `err` otherwise the next assert is useless.